CONTRIBUTING.md

```diff
@@ -89,6 +89,28 @@ To install dependencies once you have Node and yarn installed, run this in the p
 $ yarn
 ```
 
+## Adding a Plugin
+
+To create a new plugin for a third-party package, follow these steps:
+
+1. ``mkdir -p packages/datadog-plugin-<pluginname>/src``
+2. ``cp packages/datadog-plugin-kafkajs/src/index.js packages/datadog-plugin-<pluginname>/src``
+3. Edit index.js as appropriate for your new plugin
+4. ``mkdir -p packages/datadog-plugin-<pluginname>/test``
+5. ``cp packages/datadog-plugin-kafkajs/test/index.spec.js packages/datadog-plugin-<pluginname>/test``
```

Comment:
Same here:

```suggestion
5. Copy an `index.spec.js` file from another plugin to use as a starting point: `cp packages/datadog-plugin-kafkajs/test/index.spec.js packages/datadog-plugin-<pluginname>/test`
```

Reply:
Though, I don't think copying this is a good idea as this file is way too verbose. I would rather say something like "Create an `packages/datadog-plugin-<pluginname>/test/index.spec.js` file and add the necessary tests. See other plugin tests for inspiration to file structure."